HelloWorker/src/CsharpWorker.csproj

```diff
@@ -16,29 +16,29 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Improbable.WorkerSdkCsharp, Version=13.5.1.0, Culture=neutral, PublicKeyToken=null">
-      <HintPath>..\lib\improbable\sdk\13.5.1\csharp\Improbable.WorkerSdkCsharp.dll</HintPath>
+    <Reference Include="Improbable.Worker">
+      <HintPath>..\lib\improbable\sdk\14.0.0\csharp\Improbable.Worker.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Xml" />
     <Compile Include="HelloWorker.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="..\lib\improbable\sdk\13.5.1\linux64\libCoreSdkDll.so">
-      <Link>libCoreSdkDll.so</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\linux64\libimprobable_worker.so">
+      <Link>libimprobable_worker.so</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\macos64\libCoreSdkDll.dylib">
-      <Link>libCoreSdkDll.dylib</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\macos64\libimprobable_worker.dylib">
+      <Link>libimprobable_worker.dylib</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.dll">
-      <Link>CoreSdkDll.dll</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.dll">
+      <Link>improbable_worker.dll</Link>
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\lib\improbable\sdk\13.5.1\win64\CoreSdkDll.lib">
-      <Link>CoreSdkDll.lib</Link>
+    <Content Include="..\lib\improbable\sdk\14.0.0\win64\improbable_worker.lib">
```

Comment:
The `.lib` on Windows is actually not required at all for C# (as it's an DLL import library for C/C++). Therefore, you can just delete this.

Reply:
Is that the case for Mac and Linux too?

Reply:
Removed the obsolete import. :)